ecl_geometry/include/ecl/geometry.hpp

```diff
@@ -18,6 +18,7 @@
 *****************************************************************************/
 
 #include <ecl/linear_algebra.hpp>
+#include <ecl/odometry/odometry_typedefs.hpp>
```

Comment:
This is an internal include...convention puts these in quotes instead of angle brackets.